examples/features/standard/broker-plugin/pom.xml

```diff
@@ -55,6 +55,38 @@ under the License.
          <groupId>org.apache.qpid</groupId>
          <artifactId>qpid-jms-client</artifactId>
       </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-api</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-sdk</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-semconv</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-exporter-logging</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-exporter-otlp</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-exporter-jaeger</artifactId>
+      </dependency>
+      <dependency>
+         <groupId>io.opentelemetry</groupId>
+         <artifactId>opentelemetry-exporter-zipkin</artifactId>
```

Comment:
i don't what to limit the exporters that can be used.  Am passing all available exporter dependencies so that if a use chooses an exporter he/she wants, the Opentelemetry doesn't complain with `ClassNotFound` Exceptions while trying to access resources.

Reply:
Ohh.. I see.. you're light weight in dependencies...

How do you establish the connection then?

Reply:
@nbrendah I think we are in a good place to start packaging and testing this properly... we should be on a call..


There's one issue I see is that you always set a property called "count"... of course that would not work... we need to establish what property we are going to send.


and I believe the span would need a Thread Local?

Reply:
> How do you establish the connection then?

this is done the sdk during its auto-configuration.  So its upto opentelemetry to make sure the connection is set.  If any error occurs, then its a bug in auto-configuration and we have to report it back to opentelemetry-java

> we need to establish what property we are going to send.

i use count because i need to identify each span uniquely since the `getMessageId()` was returning null.  I thought using the `count` property set in the `BrokerPligin` would for a moment, create some uniqueness in spans.